### ISSUE TITLE
Can't close actual jobs nor get new jobs

### ISSUE THREAD

**Issue by pdacostaporto** [zerocracy/zerocracy.github.io#57]
I have three open jobs, currently on hold: [one](https://github.com/yegor256/0pdd/issues/221) is waiting for a new release so the reporter can see the results and close it; the [other](https://github.com/yegor256/0pdd/issues/198) [two](https://github.com/yegor256/0pdd/pull/233) have [pending](https://github.com/yegor256/0pdd/pull/232) [PRs](https://github.com/yegor256/0pdd/pull/233), but no `REV` is assigned (because there are no `REV`s on the project, I guess) to it and no one is merging it, so I'm stuck with no pay and no posibility to get new jobs.

I could resign to the jobs but that would be unfair, as the changes are made and I would get a penalty.

I guess the problem is that the project is abandoned or something (although it has funds). I think the policy should contemplate this cases. Maybe the `ARC` or someone responsible for the project should be penalised or motivated to merge the pending PRs or take care of the situation somehow and don't let the project die while it has funds and jobs in progress.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@pdacostaporto[/z](https://www.0crat.com/u/pdacostaporto) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@pdacostaporto you're right, we already have similar task to penalize `ARC` for long PRs, see https://github.com/zerocracy/farm/issues/1028 discussion.
In your case I'd suggest to ping `ARC` via all possible hannels to take some actions (add `REV` person or merge PR by itself)
